extensions/prefetch/suite.go

```diff
@@ -125,10 +125,11 @@ func (s *Suite) shouldSkipWithError(info os.FileInfo, err error) (bool, error) {
 
 func (s *Suite) fullImageName(image string) string {
 	// domain/library/name:tag
+	// or domain/library/sublibrary/name:tag
 
 	split := strings.Split(image, "/")
 	switch len(split) {
-	case 3:
+	case 4, 3:
```

Comment:
I am afraid that we can eventually run into this issue again if there will be more sublibraries.
Probably we can fix it how it is handled in CNCF distribution - https://github.com/distribution/distribution/blob/2461543d988979529609e8cb6fca9ca190dc48da/reference/normalize.go#L62

Reply:
Thanks! Fixed